.github/workflows/pull-request-lint.yml

```diff
@@ -20,10 +20,11 @@ jobs:
           # Configure which types are allowed (newline-delimited).
           # Default: https://github.com/commitizen/conventional-commit-types
           types: |
-            fix
-            feat
-            docs
-            chore
+            feature
+            bug
+            documentation
+            cleanup
+            dev
```

Comment:
This is checking the string prefix for the title, not labels. So `feat` for a `feat: xyz` title, not `feature` which is the label that gets added

Reply:
You only need to add `dev` to the existing list, and not change anything else, I believe

Reply:
yeah, i copied the wrong section